pkg/controller/metrics/metrics.go

```diff
@@ -196,6 +228,14 @@ type Calculator struct {
 // Start begins the metrics calculation loop.
 func (mc *Calculator) Start(ctx context.Context) error {
 	log.Info("started metrics calculator goroutine")
+	// Get the metrics config from hiveConfig
+	mConfig, err := readMetricsConfig()
+	if err != nil {
+		log.WithError(err).Error("cannot read metrics config")
```

Comment:
Per [slack](https://coreos.slack.com/archives/CE3ETN3J8/p1651774265655569?thread_ts=1651606872.054189&cid=CE3ETN3J8), please make this fatal.